src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,66 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_CAPACITY = 16;
+    static final double LOAD_FACTOR = 0.75;
+    private int size;
+    private MyEntry<K, V>[] values = new MyEntry[DEFAULT_CAPACITY];
 
     @Override
     public void put(K key, V value) {
-
+        boolean insert = true;
+        for (int i = 0; i < size; i++) {
+            if (values[i].key != null) {
+                if (values[i].key.equals(key)) {
+                    values[i].value = value;
+                    insert = false;
+                }
+            } else if (values[i].key == null && key == null) {
+                values[i].value = value;
+                insert = false;
+            }
+        }
+        if (insert) {
+            ensureCap();
+            values[size++] = new MyEntry<>(key, value);
+        }
```

Comment:
lets make `return;` in for loop if the value was overwritten instead of using boolean value